fivox/fieldFunctor.h

```diff
@@ -81,6 +88,9 @@ FieldFunctor< TImage >::operator()( const TPoint& point, const TSpacing& ) const
     float sum = 0.f;
     for( const Event& event : events )
     {
+        const float eventValue =
+                ( event.value - brion::MINIMUM_VOLTAGE ) * _magnitude;
+
```

Comment:
Also: before 0 meant no event. No longer true, we need a new 'unset' value for events, e.g., numeric_limits< float >::max(). cf. Source::getEvents().